build/PackageDiffIgnore.xml

```diff
@@ -8903,6 +8903,12 @@
 			<Member
 				fullName="System.Void Windows.UI.Xaml.Controls.SinglelineTextBoxView::set_SelectedTextRange(UIKit.UITextRange value)"
 				reason="Workaround for iOS 16" />
+			<Member
```

Comment:
I wonder, wouldn't we be able to keep this member as-is (albeit with a `new`) and create a new private member with a new name that has the IntPtr property?

Reply:
I guess so... but that would still be a breaking change as it would remove the `override`. Maybe the tool does not care ? OTOH any binary code that refers to the members would not be happy.

Reply:
That's fine anyways, that property is on a type that is not part of the WinUI API. I guess that it's mostly to keep the IntPtr trick inside the `SinglelineTextBoxView` class.

Reply:
So I tried implementing this:

```csharp
		public new UITextRange SelectedTextRange
		{
			get => Runtime.GetNSObject<UITextRange>(SelectedTextRangeWorkaround);
			set => base.SelectedTextRange = value;
		}

		/// <summary>
		/// Workaround for https://github.com/unoplatform/uno/issues/9430
		/// </summary>
		[Export("selectedTextRange")]
		private IntPtr SelectedTextRangeWorkaround
		{
			get
			{
				return SinglelineTextBoxView.IntPtr_objc_msgSendSuper(SuperHandle, Selector.GetHandle("selectedTextRange"));
			}
			set
			{
				if (SelectedTextRangeWorkaround != value)
				{
					SinglelineTextBoxView.void_objc_msgSendSuper(SuperHandle, Selector.GetHandle("setSelectedTextRange:"), value);
					_textBox.GetTarget()?.OnSelectionChanged();
				}
			}
		}
```

But this causes:
```
error MT4119: Could not register the selector 'selectedTextRange' of the member 'Windows.UI.Xaml.Controls.MultilineTextBoxView.get_SelectedTextRange' because the selector is already registered on the member 'get_SelectedTextRangeWorkaround'.
error MT4119: Could not register the selector 'setSelectedTextRange:' of the member 'Windows.UI.Xaml.Controls.MultilineTextBoxView.set_SelectedTextRange' because the selector is already registered on the member 'set_SelectedTextRangeWorkaround'.
```

Since` SinglelineTextBoxView` is an implementation detail, we can keep the current changes, and roll back whenever possible.

Reply:
In general (there's always exceptions) you cannot export from C#  the same selector twice (inside the same type).

Calling (managed to native) a selector is not a problem. However being called from native code requires a single entry point (method) for the selector. If there's more than one then you would not know which method would be called, hence the registrar complaining with a build error.